src/server/sams/api/consume/assets.py

```diff
@@ -50,6 +53,30 @@ def download_binary(asset_id):
     return response
 
 
+@assets_bp.route('/consume/assets/compressed_binary/<asset_ids>', methods=['GET'])
+def download_compressed_binary(asset_ids):
+    """
+    Uses asset_ids and returns the compressed
+    asset binaries zip
+    """
+    asset_ids = ast.literal_eval(asset_ids) if asset_ids else None
+    file = [get_asset_service().download_binary(asset_id) for asset_id in asset_ids]
```

Comment:
Would need to convert `asset_id` from a `str` to `bson.ObjectId` here as the `download_binary` function is expecting an instance of `bson.ObjectId`,